src/Nox.Generator/Common/TemplateScriptsBridges/NoxSolutionCodeGeneratorStateBridge.cs

```diff
@@ -14,6 +14,12 @@ public static void AddFunctions(TemplateContext context, NoxSolutionCodeGenerato
             scriptObject1.Import("CodeGeneratorStateGetForeignKeyPropertyName", new Func<NoxSimpleTypeDefinition,
                 string>((key) => codeGeneratorState.GetForeignKeyPropertyName(key.EntityTypeOptions!.Entity)));
 
+            scriptObject1.Import("CodeGeneratorPrivateFieldName", new Func<NoxSimpleTypeDefinition,
```

Comment:
we will remove the option of include scriptObject, to make the template explicit and simple... this needs to be re think to be done with script objects